developer-license.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Developer Application License
+description: Information about Discord's Developer Application License which is used to publish games.
+published: true
+date: 2020-10-15T21:50:52.089Z
+tags: 
+editor: ckeditor
+dateCreated: 2020-03-31T14:20:58.536Z
```

Comment:
Not needed and therefore pointless

Reply:
Wrong PR? Also it seems good to have just in case